modules/engines/engine-hdb/src/test/resources/registry.public.hdbview-itest/SampleHANAXSClassicView.hdbview

```diff
@@ -0,0 +1,6 @@
+//schema name to be considered when itests are run on local
```

Comment:
In the file location, do we need this `registry.public...` part? How about the following `registry.public.hdbview-itest/SampleHANAXSClassicView.hdbview` -> `hdbview-itest/SampleHANAXSClassicView.hdbview`?

Reply:
Done.